### DISCUSSION TITLE
Unsupported react version in nextjs build

### DISCUSSION THREAD

**Discussion by vitezprchal** [chakra-ui/chakra-ui#9028] [General]
### Description

Running `npm i @chakra-ui/react @emotion/react` will result in `ERESOLVE unable to resolve dependency tree`.
I'm using the latest nextjs, which is built on top of the latest version of react.

### Link to Reproduction

none

### Steps to reproduce

1. Install `npx create-next-app@latest`
2. Install `npm i @chakra-ui/react @emotion/react`

### Chakra UI Version

Latest

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by abhinav-lv** [comment]
Same error here

**Comment by segunadebayo** [comment]
Duplicate of https://github.com/chakra-ui/chakra-ui/discussions/9020

Peer-dep error from 3.0 / React 19 RC. Current `@chakra-ui/react` installs on Next 15. Update the package.
